vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/TransactionImpl.java

```diff
@@ -62,9 +63,9 @@
     }
   }
 
-  Future<Transaction> begin() {
+  Future<Transaction> begin(String startTxSql) {
```

Comment:
startTxSql should be replace by an enum as each database might use something different to do this

Reply:
also user don't want to know about the SQL for this :-)